src/common/test/componentMocking.tsx

```diff
@@ -119,17 +184,74 @@ export const renderProfileContextWrapper = async (
       () => {
         const element = getElement(selector);
         if (element && !value) {
-          if (selector.valueSelector) {
+          if (selector.valueSelector || element.nodeName === 'INPUT') {
             value = (element as HTMLInputElement).value;
+          } else {
+            value = element.textContent || undefined;
           }
-          value = element.textContent || undefined;
         }
       },
       { timeout: 150 }
     );
     return Promise.resolve(value);
   };
 
+  const triggerAction: TestTools['triggerAction'] = async selector => {
```

Comment:
Would clickElement describe this function better?

Reply:
Done.